src/Codeception/Lib/Actor/Shared/Pause.php

```diff
@@ -93,14 +93,15 @@ public function pause()
         $readline->addMapping($tput->get('key_f8'), $viewStashedFn);
         $readline->addMapping($tput->get('key_f10'), $clearStashFn);
 
-        $output->writeln("  <comment>Execution PAUSED, starting interactive shell...</comment>");
+        $output->writeln("  <comment>Execution PAUSED, starting Interactive Shell...</comment>");
         $output->writeln("  Type in commands to try them:");
         $output->writeln("  - <info>ENTER</info> to continue");
         $output->writeln("  - <info>TAB</info> to auto-complete");
         $output->writeln("  - <info>F5</info> to stash a command");
         $output->writeln("  - <info>F6</info> to toggle auto-stashing of successful commands");
         $output->writeln("  - <info>F8</info> to view stashed commands");
         $output->writeln("  - <info>F10</info> to clear stashed commands");
+        $output->writeln("  - <info>exit</info> to leave the Interactive Shell");
```

Comment:
Based on the code below, entering `exit` has the same effect as pressing ENTER on empty line.

I think that the line which must be fixed is `Type in commands to try them`, because the only command that can be typed is `exit`, all others are keyboard keys that can be pressed.

Reply:
Yeah, I only noticed that afterwards ;-) I updated it now.
2 questions:
* Then why is `exit` in the code at all? Is there a use-case where ENTER may not work? If not, I'll delete the `exit` line again.
* Just noticed that Backspace is not working, so there's no way to fix a typo?! Is this a limitation of `hoa/console`?

Reply:
Please delete the `exit` line.

I can't answer question about hoa/console, because I am not familiar with it.

There is some ambiguity in the current text that it doesn't tell to press ENTER after typing command.

Reply:
> Please delete the exit line.

Done :-)

> There is some ambiguity in the current text that it doesn't tell to press ENTER after typing command.

I guess that's clear to anybody :-)

Reply:
Just found a way to correct errors: `UP` key!